### PR TITLE
Adds collector for Scribe.

### PR DESCRIPTION
**Pull request by GeorgeSirois**
@steveniemitz @ThanosBaskous

### CODE REVIEW

File: src/collectors/scribe/scribe.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        Get metrics from a specific port.
+        """
+        command = [self.config['bin'], 'counters', str(port)]
+        try:
+            proc = subprocess.Popen(command, stdout=subprocess.PIPE)
+            out, _ = proc.communicate()
+
+            return self._parse_raw_metrics(out)
+        except Exception:
+            self.log.exception('Unable to collect disk usage.')
```
Comment (steveniemitz) [inline src/collectors/scribe/scribe.py]: I assume this is from a copy/pasta?  Maybe rename to "Unable to collect scribe metrics"?

File: src/collectors/teamcity/test/testteamcity.py [hunk 2]
```diff
@@ -22,7 +20,7 @@
 
 ################################################################################
 
-class TestMesosCollector(CollectorTestCase):
+class TestTeamCityCollector(CollectorTestCase):
```
Comment (steveniemitz) [inline src/collectors/teamcity/test/testteamcity.py]: Thanks!

File: src/collectors/scribe/scribe.py (hunk 1 above)
Comment (GeorgeSirois) [inline src/collectors/scribe/scribe.py]: Oops, yeah that was cribbed from another collector

**Comment by steveniemitz** [comment]
:+1:

**Pull request merged**